sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_server_events_v1/query.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+  SPLIT(jsonPayload.logger, "-")[
+    OFFSET(1)
+  ] AS fxa_server,  -- example expected input: fxa-auth-server
```

Comment:
This is only being formatted split across multiple lines due to the trailing comment.  If you put the comment on the preceding line it would format nicer.
```suggestion
  -- example logger value: fxa-auth-server
  SPLIT(jsonPayload.logger, "-")[OFFSET(1)] AS fxa_server,  
```

Reply:
made the tweak